velox/vector/tests/VectorMakerTest.cpp

```diff
@@ -458,6 +459,42 @@ TEST_F(VectorMakerTest, arrayVectorFromJson) {
   }
 }
 
+TEST_F(VectorMakerTest, arrayOfArraysFromJson) {
+  auto arrayVector = maker_.arrayOfArraysFromJson<int32_t>(
+      {"[[1, 2], [2, 3, 4], [null, 7]]", "[[1, 3, 7, 9], []]", "[]", "[null]"});
```

Comment:
add trailing comma for readability

perhaps, add one more case "[[1, 2, null], [], null]"

Reply:
added testcase in new commit